ext/sysvmsg/php_sysvmsg.h

```diff
@@ -27,4 +27,11 @@ extern zend_module_entry sysvmsg_module_entry;
 
 #endif /* HAVE_SYSVMSG */
 
+/* In order to detect MSG_EXCEPT use at run time; we have no way
+ * of knowing what the bit definitions are, so we can't just define
+ * out own MSG_EXCEPT value. */
```

Comment:
Let's fix that typo right away:
```suggestion
 * our own MSG_EXCEPT value. */
```